dali/kernels/test/tensor_test_utils.h

```diff
@@ -219,19 +222,21 @@ if_iterable<Collection, void> UniformRandomFill(
   Fill(std::forward<Collection>(c), generator);
 }
 
-template <typename DataType, int ndim, typename RandomGenerator>
+template <typename StorageBackend, typename DataType, int ndim, typename RandomGenerator>
 void UniformRandomFill(
-    const TensorView<StorageCPU, DataType, ndim> &tv,
+    const TensorView<StorageBackend, DataType, ndim> &tv,
     RandomGenerator &generator,
     same_as_t<DataType> lo, same_as_t<DataType> hi) {
+  static_assert(is_cpu_accessible<StorageBackend>::value, "Function available only for CPU-accessible TensorView backend");
```

Comment:
This message is convoluted.
```suggestion
  static_assert(is_cpu_accessible<StorageBackend>::value, "This function requires a CPU-acessible tensor.");
```